doc/index.rst

```diff
@@ -95,6 +95,21 @@ RSS
 **rss/channel/generator**
     A string indicating the program used to generate the channel. (e.g. MightyInHouse Content System v2.3).
 
+**rss/channel/language**
+    Podcast language.
+
+**rss/channel/itunes:author**
+    The group responsible for creating the show.
+
+**rss/channel/itunes:owner**
+    The podcast owner contact information.
+
+**rss/channel/itunes:explicit**
+    Indicates whether podcast contains explicit material.
+
+**rss/channel/itunes:new-feed-url**
+    The new podcast RSS Feed URL.
```

Comment:
This should probably be exposed as a new URL and not `new_feed_url`, I'm sure other podcast extensions have similar "redirect" kind of things (e.g. [`redirect/newLocation`](https://www.rssboard.org/redirect-rss-feed)) and both of these should map to the same thing.

Reply:
Done